docs/workforce-management-framework/topics/pro-enabling-logging.adoc

```diff
@@ -0,0 +1,27 @@
+= Enabling logging
+
+By default, all {WFM-RC-NameLong} framework modules will not log any information to the console.
+Framework users can enable logger by using one of the provided implementations or the
+custom implementation that will extend our interface.
+
+Supported implementations:
+
+|====
+| Implementation | Usage
+
+|BunyanLogger |Implements Bunyan and is used for server side logging
+|ClientLogger |Implements LogLevel and is used for client (web and mobile) logging
+|====
+
```

Comment:
No.. We should take different angle here and in all documentations. 
We should provide mostly why this modules is used, what value it gives to user and how user should interact with it. This pattern applies to all module docs. 
We should minimize source code as you see we already changed a lot of names etc.

Things getting outdated so fast that we may change them 5-6 times before releasing docs.